addons/web/static/src/core/dialog/dialog.js

```diff
@@ -21,6 +21,9 @@ export class Dialog extends Component {
             { altIsOptional: true }
         );
         useSubEnv({ inDialog: true });
+        this.bodyTemplate = "web.DialogBodyDefault";
+        this.footerTemplate = "web.DialogFooterDefault";
+        this.close = this.close.bind(this);
```

Comment:
we do this because of https://github.com/odoo/owl/issues/862
The Portal of the Dialog is not the parent of the Components we use in the t-call ; so instead we pass the close function if it is needed by Components down the line